src/webhook/upgrade_operator/fetch_upgradeconfig.py

```diff
@@ -0,0 +1,20 @@
+import os
+from kubernetes import client, config
+from kubernetes.client.rest import ApiException
+
+# Use KUBECONFIG from pod else consume from local ~/.kube/config
+if 'KUBERNETES_PORT' in os.environ:
+    config.load_incluster_config()
+else:
+    config.load_kube_config()
+
+
+def get_upgradeconfig_cr():
+    custom_api = client.CustomObjectsApi()
+    try:
+        ugpradeconfig_resource = custom_api.list_cluster_custom_object(
+            'upgrade.managed.openshift.io', 'v1alpha1', 'upgradeconfigs')
+        return ugpradeconfig_resource
+    except ApiException as e:
+        print(
+            "Exception in fetching upgrade.managed.openshift.io/v1alpha1 resource: %s\n" % e)
```

Comment:
This will implicitly return `None`. That would be better to make explicit.

Reply:
Sorry, I didn't quite understand this comment. Could you elaborate? If you are referring to whether the script the method does return a valid dict object or not, it does. I am able to fetch the value in the calling script which uses this helper script.

Reply:
Ok, I think I understand your point. I will add checks to return `None` explicitly. Thanks a lot @2uasimojo !

Reply:
Done